src/scipp/html/table.py

```diff
@@ -0,0 +1,230 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+import numpy as np
+from typing import List, Optional, Union
+from .. import DataArray, Dataset, Variable
+from ..typing import VariableLike
+from .. import DType
+
+CENTER = 'style="text-align: center;"'
+WITH_BORDER = 'style="border-left:1px solid #a9a9a9;"'
+CENTER_BORDER = 'style="text-align: center; border-left:1px solid #a9a9a9;"'
```

Comment:
Looks good. But I have some comments about the style:

- Tables of a single variable have a black border on the left because they are treated like datasets without coords. This looks a bit odd.
- The separators between dataset items don't extend to the row of dataset names. Is this intentional?
- It might help to put a line between the header and the actual data (like pandas).

Reply:
> Tables of a single variable have a black border on the left because they are treated like datasets without coords. This looks a bit odd.

Yes, it was the simplest thing to leave it as is. But I can try to fix if you want.

> The separators between dataset items don't extend to the row of dataset names. Is this intentional?

Yes, I thought it looked cleaner without the lines going all the way at the top. But we can change.
![Screenshot at 2022-05-30 14-15-55](https://user-images.githubusercontent.com/39047984/170991245-89c8cfc4-a5f5-4558-aa0e-339fce488692.png)

> It might help to put a line between the header and the actual data (like pandas).

:+1:

Reply:
> Yes, it was the simplest thing to leave it as is. But I can try to fix if you want.

Depends on how much extra code this requires. It should be rare to use `table` for variables...

> Yes, I thought it looked cleaner without the lines going all the way at the top. But we can change.

I find both a bit strange because the names are so short. But I prefer the left because it makes it clearer that 'a' and 'b' are part of the table.